en/frames_autogyro/README.md

```diff
@@ -0,0 +1,12 @@
+# Autogyro Frame Builds
+
+[Autogyro](https://en.wikipedia.org/wiki/Autogyro) is a type of [rotor-wing](https://en.wikipedia.org/wiki/Rotorcraft). Compared to other designs it possesses following advantages:
+
+  * Ability to take off and land using only a very short runway.

```

Comment:
Compared to a fixed wing, but not a multicopter :-)

Reply:
Ok; Would it be better to do a list of properties (or flight characteristics) from this comparison, which autogyro has?
I don't want to compare it to a multicopter, because it's a completely different drone for a different purpose.

Reply:
solved.